src/asyncUtils.ts

```diff
@@ -1,28 +1,35 @@
 import { act } from 'react-test-renderer'
 
-function createTimeoutError(utilName, { timeout }) {
-  const timeoutError = new Error(`Timed out in ${utilName} after ${timeout}ms.`)
-  timeoutError.timeout = true
+export interface WaitOptions {
+  interval?: number
+  timeout?: number
+  suppressErrors?: boolean
+}
+
+class TimeoutError extends Error {
+  timeout = true
```

Comment:
I don't think this field is being used anymore now that `error instanceof TimeoutError` works